packages/dds/sequence/src/sharedSequence.ts

```diff
@@ -31,7 +31,7 @@ export class SubSequence<T> extends BaseSegment {
 	public static is(segment: ISegment): segment is SubSequence<any> {
 		return segment.type === SubSequence.typeString;
 	}
-	public static fromJSONObject<U>(spec: Serializable) {
+	public static fromJSONObject<U>(spec: any) {
```

Comment:
`unknown`?

Reply:
`SubSequence` is deprecrated - keeping type the same => `any`.